assets/scripts/segments/info.json

```diff
@@ -691,7 +691,8 @@
     "defaultWidth": 6,
     "variants": [
       "direction",
-      "bike-asphalt"
+      "bike-asphalt",
+      "bike-lane-elevation"
```

Comment:
What happens if this is just `elevation`? (See comment below.)

Reply:
Responded to previous comment.